client/hbs/not-found.hbs

```diff
@@ -1 +1,15 @@
-Ya dun goofed. Go <a href="/">home</a>?
+<html>
+{{> head.hbs}}
+
+<body>
+	{{> header.hbs}}
+	<content-row>
+		<main>
+			<p>Hmmm... looks like we couldn't find that.</p>
+			<p>If we sent you here, please <a href="mailto:webmaster@dttw.tech">let us know</a>.</p>
```

Comment:
I know the old one didn't do this either, but there should be a decently prominent `404` somewhere on there